perl/buffers.pl

```diff
@@ -1298,8 +1300,7 @@ sub build_buffers
             }
             else
             {
-                my $indent = "";
-                $indent = ((" " x length($buffer->{"number"}))." ") if (($position eq "left") || ($position eq "right"));
+                my $indent = (" " x length($buffer->{"number"}))." ";
```

Comment:
In the commit 0149cb9be5f1e2b50de9d54694f4a843a8c378b8, I disabled the indent on top/bottom position, and I think we should keep that with a small change. The indent should be enabled on top/bottom ONLY if the filling is not "horizontal". In other words, the indent should be enabled everywhere, except if you have position top/bottom with horizontal filling.